backends/intel_gpu/CMakeLists.txt

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+cmake_minimum_required(VERSION 3.10)
+
+project(paddle-custom-intel-gpu CXX C)
+
+##########################################
+
+
+if(NOT DEFINED ONEAPI_PATH)
+set(ONEAPI_PATH "$ENV{HOME}/intel/oneapi")
+endif()
+
+if(NOT EXISTS "${ONEAPI_PATH}")
+message(FATAL_ERROR "Can't find openapi directory ${ONEAPI_PATH}")
+endif()
+
+message(STATUS "OneAPI DIR : ${ONEAPI_PATH}")
+
+if(NOT DEFINED ONEAPI_COMPILER_DIR)
+#set(ONEAPI_COMPILER_DIR "compiler/2022.1.0")
+set(ONEAPI_COMPILER_DIR "compiler/latest")
+endif()
+
+if(NOT EXISTS "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}")
+message(FATAL_ERROR "Can't find directory ${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}")
+endif()
+
+if(NOT DEFINED ENV{CONDA_PREFIX})
+message(FATAL_ERROR "Please install python miniconda ")
+endif()
+
+if(NOT DEFINED CND_INSTALL_DIR)
+    set(CND_INSTALL_DIR "$ENV{CONDA_PREFIX}")
+endif()
+
+if(NOT EXISTS "${CND_INSTALL_DIR}")
+ message(FATAL_ERROR "Can't find conda directory ${CND_INSTALL_DIR}")
+endif()
+
+if(NOT DEFINED ENV{DPL_ROOT})
+	message(FATAL_ERROR "DPL_ROOT ist not set !!! " $ENV{DPL_ROOT})
+endif()
+message(STATUS "Conda DIR : ${CND_INSTALL_DIR}")
+
+# if(NOT DEFINED CND_ENV)
+# set(CND_ENV "$ENV{CONDA_DEFAULT_ENV}")
+# endif()
+
+# message(STATUS "CONDA default env => ${CND_ENV}")
+
+#set(CND_PYTHON "${CND_INSTALL_DIR}/envs/${CND_ENV}/lib/python3.9")
+set(CND_PYTHON "${CND_INSTALL_DIR}/lib/python3.9")
+
+
+if(NOT EXISTS "${CND_PYTHON}")
+   message(FATAL_ERROR "Can't find conda python directory ${CND_PYTHON}, python3.9 required!" )
+endif()
+
+
+
+#############################
+
+set(DPCPP_COMPIER_PATH "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}/linux/bin/dpcpp")
+set(ONEAPI_SYCL_INCLUDE "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}/linux/include/sycl/")
+set(ONEAPI_SYCL_LIBDIR "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}/linux/lib/")
+set(CMAKE_CXX_COMPILER "${DPCPP_COMPIER_PATH}")
+##########################################
+#set(ONEDNN_INC "${PADDLE_BUILD_DIR}/third_party/install/mkldnn/include")
+#set(ONEDNN_LIB "${PADDLE_BUILD_DIR}/third_party/install/mkldnn/lib")
+##########################################
+set(PADDLE_PLUGIN_DIR  "${CND_PYTHON}/site-packages/paddle-plugins/")
+set(PADDLE_INC_DIR     "${CND_PYTHON}/site-packages/paddle/include/")
+set(PADDLE_LIB_DIR     "${CND_PYTHON}/site-packages/paddle/fluid/")
```

Comment:
Please use the PADDLE_INC_DIR and PADDLE_LIB_DIR found from paddle.cmake, this two variables will be override by "include(paddle)" below.

Reply:
done